src/check-deployer.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Deployer;
+
+desc('Check currently used deployer path');
+task('s24:check-deployer', function () {
+
+    list($scriptPath) = get_included_files();
```

Comment:
I find this difficult to read, I think it assigns the first item in the array returned by get_included_files() to $scriptPath. I normally see list used when a function returns a fixed array of multiple things.

In this example this would be clearer: 
`$scriptPath = get_included_files()[0];`

Can I check, do you want to only check the first item returned by get_included_files() - is this always going to be Deployer? I am not sure that is the case.

I also think this is the path you want to check is in the include path:
`
__DIR__ . '/vendor/deployer/deployer/bin/dep'
`

I don't think there's a need to output a message if the user is using local deployer. I'd just error if the user is not to keep things succinct.

I would move the error message about please run using local Deployer can be moved to the exception call.